src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -168,6 +170,49 @@ private IEnumerable<CompletionItem> GetResourceTypeCompletions(SemanticModel mod
                 .ToList();
         }
 
+        private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath) 
+            || Path.GetDirectoryName(model.SyntaxTree.FileUri.LocalPath) is not string cwd
```

Comment:
The language server cannot make any assumptions about the FileUri being an actual physical file, because it doesn't have to be one. Any usage of APIs with file system interaction (`Path.XYZ`, `File.Exists`, `Directory.Exists`) should be behind the `IFileResolver` interface.